packages/react-composites/tests/browser/chat/app/fake-back-end/Model.ts

```diff
@@ -48,6 +48,23 @@ export class Model {
       });
     }
   }
+
+  public initializeMessages(threadId: string, messages: BasicChatMessage[]) {
+    messages.forEach((message) => {
+      this.checkedGetThread(message.sender, threadId);
+      this.threadMap[threadId].thread.messages = [
+        ...this.threadMap[threadId].thread.messages,
+        {
+          id: `${this.threadMap[threadId].thread.messages}`,
+          sequenceId: `${this.threadMap[threadId].thread.messages}`,
```

Comment:
this is a little confusing to read here, can you add a fn to the thread interface that is pushMessages(messages) or similar that handles updating the id, sequenceId and other default values.